apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -148,4 +147,16 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     }
     return text;
   }
+
+  private _generateSnippetText(snippets: string[]): string {
+    let text: string = '\n\n#### Examples\n';
```

Comment:
>\n [](start = 24, length = 2)

The way I usually do this is to create a string array with each index as a line, and then run `.join('\n\)` to generate the full text. It's easier to read.

Reply:
I'll give it shot.